### ISSUE TITLE
timestampable listener not active/not working

### ISSUE THREAD

**Issue by alessandro-podo** [doctrine-extensions/DoctrineExtensions#2501]
I am not sure why and dont know how to debug this. 

When i use the "Gedmo\Timestampable\Traits\TimestampableEntity;" i always got the Form Error, that the Value can't be null. 

The "TimestampableListener" isnt active. Checked via "console debug:event-dispatcher  |grep Stof"

This 2 Listener i can see: 
```
#21     Stof\DoctrineExtensionsBundle\EventListener\LoggerListener::onKernelRequest()                  0         
#22     Stof\DoctrineExtensionsBundle\EventListener\BlameListener::onKernelRequest()                   0    
```

My Config: 
```yaml
stof_doctrine_extensions:
    default_locale: de_DE
    orm:
        lokal: 
            timestampable: true
            loggable: true
            blameable: true
```

**Comment by alessandro-podo** [comment]
I forgot to check if it works without a form.

The entity is filled with the new values when the submit is complete and the flush occurs. Before that,  the validator has already thrown the error.

**Comment by thewrath** [comment]
I face the same problem but only in prod environment, if I remove the following config : 
`when@prod:
    doctrine:
        orm:
            auto_generate_proxy_classes: false
            query_cache_driver:
                type: pool
                pool: doctrine.system_cache_pool
            result_cache_driver:
                type: pool
                pool: doctrine.result_cache_pool

    framework:
        cache:
            pools:
                doctrine.result_cache_pool:
                    adapter: cache.app
                doctrine.system_cache_pool:
                    adapter: cache.system
`
it's working again. I'll see if an update or a configuration change can fix it.

**Comment by thewrath** [comment]
@alessandro-podo it seems that the problem has been fixed in version 3.8

**Comment by drekinov** [comment]
it seems mine #2565 is duplicate of that issue.
i am with 3.10 and get SQL error because probably listener is not active for some reason only in prod env.
======
workaround which i just come up is to have created_at in constructor of entity and not relying on extension.
it just started to fail more often and cause production issues
